test/config/integration/server_v2.json

```diff
@@ -0,0 +1,818 @@
+{
```

Comment:
Should we convert these to YAML since they are checked in files that will be used by folks? I wouldn't suggest doing it by hand, but maybe some CLI or Web-based JSON->YAML converter.

Reply:
Sure.   It doesn't automatically convert because of all the {{}}s but I swapped them out, converted, and then (I think) swapped them back correctly.  Oh and a bunch more updates to the scripts which want to parse yaml.  Now they read yaml and split out json because dumping yaml caused some weird parsing error and I think this is v2 enough for now. :-)

I think given this isn't going to get consensus to land this morning, and next week is a holiday week for folks in the states I'm going to make this a test-only change, email envoy-announce, and flag flip on the 9th.  Description updated accordingly.